elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/AggregationDsl.scala

```diff
@@ -42,6 +42,7 @@ trait AggregationDsl extends DslCommons with QueryDsl {
                               size: Option[Int], shardSize: Option[Int],
                               hint: Option[String] = None,
                               name: Option[String] = None,
+                              order: Option[SortOrder] = None,
```

Comment:
Yeah, I almost did that, but it seemed fitting to have nested aggs at the end. I'll be happy to change it though.